Added healthcheck timeout/start-period/retries

micahkemp-splunk opened the pull request:
#12

acharlieh reviewed (approved):
Only thing I wonder about... if we could define a reasonable default health check interval in seconds too, then the default command could have only one if condition of having a specified series or if someone wanted a custom health check command they wouldn’t have an error if they forgot to also specify interval as well

But that’s unrelated existing issue so approving

micahkemp-splunk commented:
> c

I added this default.  We previously used it as a condition for defining using the default healthcheck.  I've removed that condition, and other references to needing to set it explicitly.

In our inventory we had it set in `group_vars`, so it has some type of default there, which can now be removed.e
